rust/Cargo.toml

```diff
@@ -1,28 +1,10 @@
-[package]
-name = "tvm"
-version = "0.1.0"
-license = "Apache-2.0"
-description = "TVM Rust runtime"
-repository = "https://github.com/dmlc/tvm"
-readme = "README.md"
-keywords = ["tvm", "nnvm"]
-categories = ["api-bindings", "science"]
-authors = ["Nick Hynes <nhynes@berkeley.edu>"]
-
-[features]
-default = ["nom/std"]
-sgx = ["nom/alloc"]
-
-[dependencies]
-bounded-spsc-queue = "0.4.0"
-error-chain = { version = "0.12.0", default-features = false }
-itertools = "0.7.8"
-lazy_static = "1.1.0"
-ndarray = "0.11.2"
-nom = {version = "4.0.0", default-features = false }
-serde = "1.0.59"
-serde_derive = "1.0.79"
-serde_json = "1.0.17"
-
-[target.'cfg(not(target_env = "sgx"))'.dependencies]
-num_cpus = "1.8.0"
+[workspace]
+members = [
+			"runtime",
+			"runtime/tests/test_tvm_basic",
+			"runtime/tests/test_nnvm",
+			"frontend", 
+			"frontend/tests/basics", 
+			"frontend/tests/callback",
+			"frontend/examples/resnet"
+			]
```

Comment:
Thanks for the suggestion! I tried but since `tests/basics` and `examples/resnet` need to be their own crate as well with custom `build.rs` I couldn't add them. For `tests/callback` per our discussion before, each [global function registration](https://discuss.tvm.ai/t/are-global-functions-need-to-be-accessed-in-separate-processes/1075/2) needs to be run separately, then I should exclude that as well. So I think this is the optimal setup for now.